exams/migrations/0008_populate_exam_run.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-04-24 19:10
+from __future__ import unicode_literals
+from datetime import datetime
+
+from django.db import migrations
+
+PILOT_SERIES_CODE = 'PILOT'
+PILOT_SCHEDULE_START_DATE = datetime(2017, 3, 6)
+PILOT_SCHEDULE_END_DATE = datetime(2017, 3, 30)
+PILOT_ELIGIBILITY_START_DATE = datetime(2017, 3, 18)
+PILOT_ELIGIBILITY_END_DATE = datetime(2017, 3, 31)
+
+
+def populate_pilot_runs(apps, schema_editor):
+    """Generate pilot ExamRuns"""
+    ExamRun = apps.get_model('exams', 'ExamRun')
+    ExamAuthorization = apps.get_model('exams', 'ExamAuthorization')
+    runs_by_course_id = {}
+
+    for exam_auth in ExamAuthorization.objects.filter(exam_run__isnull=True).iterator():
+        if exam_auth.course_id not in runs_by_course_id:
+            runs_by_course_id[exam_auth.course_id] = ExamRun.objects.create(
+                course=exam_auth.course,
+                exam_series_code=PILOT_SERIES_CODE,
+                date_first_schedulable=PILOT_SCHEDULE_START_DATE,
+                date_last_schedulable=PILOT_SCHEDULE_END_DATE,
+                date_first_eligible=PILOT_ELIGIBILITY_START_DATE,
+                date_last_eligible=PILOT_ELIGIBILITY_END_DATE,
+            )
+        exam_auth.exam_run = runs_by_course_id[exam_auth.course_id]
+        exam_auth.save(updated_fields=('exam_run',))
```

Comment:
I got an exception on this line:

    Operations to perform:
      Apply all migrations: admin, auth, authtoken, cms, contenttypes, courses, dashboard, ecommerce, exams, financialaid, grades, mail, profiles, roles, search, sessions, social_django, taggit, wagtailadmin, wagtailcore, wagtaildocs, wagtailembeds, wagtailforms, wagtailimages, wagtailredirects, wagtailsearch, wagtailusers
    Running migrations:
      Applying exams.0008_populate_exam_run.../usr/local/lib/python3.5/site-packages/django/db/models/fields/__init__.py:1430: RuntimeWarning: DateTimeField ExamRun.date_first_schedulable received a naive datetime (2017-03-06 00:00:00) while time zone support is active.
      RuntimeWarning)
    /usr/local/lib/python3.5/site-packages/django/db/models/fields/__init__.py:1430: RuntimeWarning: DateTimeField ExamRun.date_last_schedulable received a naive datetime (2017-03-30 00:00:00) while time zone support is active.
      RuntimeWarning)
    Traceback (most recent call last):
      File "./manage.py", line 14, in <module>
        execute_from_command_line(sys.argv)
      File "/usr/local/lib/python3.5/site-packages/django/core/management/__init__.py", line 367, in execute_from_command_line
        utility.execute()
      File "/usr/local/lib/python3.5/site-packages/django/core/management/__init__.py", line 359, in execute
        self.fetch_command(subcommand).run_from_argv(self.argv)
      File "/usr/local/lib/python3.5/site-packages/django/core/management/base.py", line 294, in run_from_argv
        self.execute(*args, **cmd_options)
      File "/usr/local/lib/python3.5/site-packages/django/core/management/base.py", line 345, in execute
        output = self.handle(*args, **options)
      File "/usr/local/lib/python3.5/site-packages/django/core/management/commands/migrate.py", line 204, in handle
        fake_initial=fake_initial,
      File "/usr/local/lib/python3.5/site-packages/django/db/migrations/executor.py", line 115, in migrate
        state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)
      File "/usr/local/lib/python3.5/site-packages/django/db/migrations/executor.py", line 145, in _migrate_all_forwards
        state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
      File "/usr/local/lib/python3.5/site-packages/django/db/migrations/executor.py", line 244, in apply_migration
        state = migration.apply(state, schema_editor)
      File "/usr/local/lib/python3.5/site-packages/django/db/migrations/migration.py", line 129, in apply
        operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
      File "/usr/local/lib/python3.5/site-packages/django/db/migrations/operations/special.py", line 189, in database_forwards
        self.code(from_state.apps, schema_editor)
      File "/src/exams/migrations/0008_populate_exam_run.py", line 32, in populate_pilot_runs
        exam_auth.save(updated_fields=('exam_run',))
    TypeError: save() got an unexpected keyword argument 'updated_fields'